src/Screens/Explore/Reviews.tsx

```diff
@@ -133,20 +134,10 @@ const Reviews: FunctionComponent<ReviewProps> = ({ navigation, route }) => {
 
                   <View style={styles.nameSourceContainer}>
                     <Text style={styles.profileName}>
-                      {item.user.first_name}
+                      {item.user.display_name}
                     </Text>
                     <View
-                      style={[
-                        styles.activityTypeContainer,
-                        {
```

Comment:
Um, this piece of code was for that label issue where some of the labels became very long.

Reply:
fixed below with flex-start